.eslintrc

```diff
@@ -0,0 +1,30 @@
+{

```

Comment:
copy the `.eslintrc.yml` file from the chill repo instead. https://github.com/leapfrogtechnology/chill/blob/master/.eslintrc.yml